pgx-utils/src/lib.rs

```diff
@@ -7,29 +7,15 @@ All rights reserved.
 Use of this source code is governed by the MIT license that can be found in the LICENSE file.
 */
 
-use crate::{pg_config::PgConfig, sql_entity_graph::PositioningRef};
-use eyre::{eyre, WrapErr};
-use owo_colors::OwoColorize;
+use crate::sql_entity_graph::PositioningRef;
 use proc_macro2::{TokenStream, TokenTree};
 use quote::{format_ident, quote, ToTokens, TokenStreamExt};
-use serde_json::value::Value as JsonValue;
-use std::{
-    collections::HashSet,
-    path::PathBuf,
-    process::{Command, Stdio},
-    str::FromStr,
-};
```

Comment:
I know it doesn't look like it but this is probably one of the most important improvements here.

Reply:
Yeah, based on what you said earlier this is huge.